src/CounterInterceptor/WorkerCounter.cs

```diff
@@ -0,0 +1,91 @@
+namespace TemporalioSamples.CounterInterceptor;
+
+using System.Numerics;
+
+public static class WorkerCounter
+{
+    public const string NumberOfWorkflowExecutions = "numOfWorkflowExec";
```

Comment:
While I personally prefer camel for readability (and, for constants like this upper camel -- but again, that's personal), in the learn samples we consistently use screaming snake for shared constants.

new TemporalWorkerOptions(taskQueue: "MONEY_TRANSFER_TASK_QUEUE")

Feel free to check with other docs folk

Reply:
Removed the constants based on Chad's recommendation